python/interpret_community/__init__.py

```diff
@@ -2,7 +2,11 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # ---------------------------------------------------------
 
-"""Module for interpreting, including feature and class importances for blackbox, greybox and whitebox models."""
+"""Module for interpreting, including feature and class importance for blackbox, greybox and whitebox models.
```

Comment:
Do we want to change whitebox to glassbox?

Reply:
great catch, also updated some other docs for whitebox that was incorrect